cypress/utils/components/cru-resource.po.ts

```diff
@@ -206,9 +212,52 @@ export default class CruResourcePo extends PagePo {
     this.save()
   }
 
+  // make sure that current page is the list page.
+  public search(queryString: any) {
+    cy.get('.sortable-table-header .search-box').should('be.visible');
+    cy.get(this.searchInput).type(queryString);
```

Comment:
cy.get(this.searchInput).clear().type(queryString);

Reply:
Done.